src/components/ImageUploader/src/ImageSelection.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+	<div
+		:class="{ [$s.ImageSelectionContainer]: true, [$s.ImageSelectionContainerError]: hasError }"
```

Comment:
Should be multiline but ideally something caught by ESLint.

Will look into it